Source/Processors/Editors/ChannelSelector.cpp

```diff
@@ -580,6 +581,7 @@ int ChannelSelector::getDesiredWidth()
 void ChannelSelector::buttonClicked(Button* button)
 {
     //checkChannelSelectors();
+    ListSliceParser* sp = new ListSliceParser("");

```

Comment:
There we will get memory leak.

To get ride of this, the better way would be to define appropriate parsing method as static and just call it.
We don't need to create instance of this class.